src/service/search/NativeSearchService.ts

```diff
@@ -77,29 +71,41 @@ class NativeSearchService implements AbstractSearchService {
     }
 
     async createSearchResult(
-            term: string,
-            details: IPlayerDetails,
-            sizeFactor: number,
-            synonym?: Synonym
-        ): Promise<IPlayerSearchResult> {
-            return {
-                number: 0,
-                title: details.title,
-                channel: details.channel || '',
-                pid: details.pid,
-                request: {
-                    term,
-                    line: term,
-                },
-                episode: details.episode,
-                pubDate: details.firstBroadcast ? new Date(details.firstBroadcast) : undefined,
-                series: details.series,
-                type: details.type,
-                size: details.runtime ? Math.floor(details.runtime * 60 * sizeFactor) : undefined,
-                nzbName: await createNZBName(details, synonym),
-                episodeTitle: details.episodeTitle,
-            };
-        }
+        term: string,
+        details: IPlayerDetails,
+        sizeFactor: number,
+        synonym?: Synonym
+    ): Promise<IPlayerSearchResult> {
+        return {
+            number: 0,
+            title: details.title,
+            channel: details.channel || '',
+            pid: details.pid,
+            request: {
+                term,
+                line: term,
+            },
+            episode: details.episode,
+            pubDate: details.firstBroadcast ? new Date(details.firstBroadcast) : undefined,
```

Comment:
Might be worth using the release date the new endpoint provides, seemed more accurate in my initial tests.